docs/lib/datastore/fragments/js/other-methods.md

```diff
@@ -0,0 +1,19 @@
+## Clear
+
+To clear local data from DataStore, use the `clear` method:
+
+```js
```

Comment:
I would put a note about `.clear()` being important to use for shared device scenarios or where you need to purge the local on-device storage of records for security/privacy concerns.